core/local/local.go

```diff
@@ -98,7 +98,7 @@ func NewExecutionScheduler(runner lib.Runner, logger *logrus.Logger) (*Execution
 		logger:  logger,
 		options: options,
 
-		initProgress:    pb.New(pb.WithConstLeft("Init")),
+		initProgress:    pb.New(pb.WithConstLeft(" Init")),
```

Comment:
hmm this extra space seems superfluous :confused: I thought the extra space before ` Run` was so it has exactly the same number of characters `Init` already has without padding? Though looking at how it looks like, even that seems superfluous?

Reply:
It was a few weeks since I changed this, but I think it was to make this consistent for both `cloud` and `run`, but you're right that it looks off.

Reverted in 20b988aa.